examples/advanced/main.tf

```
@@ -66,6 +66,9 @@ resource "betteruptime_monitor" "playwright" {
     });
   EOT
   request_timeout   = 60
+  environment_variables = {
+    PASSWORD : "passw0rd"
```

💄 consistent with test cases / other examples

```suggestion
    PASSWORD = "passw0rd"
```

---

💄 It would also be great to use the variable using process.env.PASSWORD in the playwright script

---

> 💄 It would also be great to use the variable using process.env.PASSWORD in the playwright script

I see no "natural" way how to use the password (or similar) on betterstack.com, so I'm thinking of a few options:

  1. Change the example to check something like [https://httpbin.org/basic-auth/{user}/{passwd}](https://httpbin.org/#/Auth/get_basic_auth__user___passwd_). This would demonstrate the intended use, but it's not very realistic and we will unnecessarily spam httpbin.org with requests. This would be similar with any other website.
  2. Change the variable to something like `TITLE` and keep the current example (modulo using the variable). This doesn't show the intended use + also not very realistic.
  3. Keep the example as it is.
  
@PetrHeinz What do you think? Maybe you see some other option I don't?

---

I'd either keep it as is, use move the title to the secret, or use secret `EMAIL = "hello@betterstack.com"` and expect it on the page (it's in the footer).

---

> use secret EMAIL = "hello@betterstack.com" and expect it on the page (it's in the footer)

Thanks, done that.